lib/infra/17-cadvisor.yml

```diff
@@ -86,7 +86,7 @@ spec:
     spec:
       automountServiceAccountToken: false
       containers:
-      - image: k8s.gcr.io/cadvisor:v0.30.2
+      - image: k8s.gcr.io/cadvisor:v0.39.2
```

Comment:
Looks like they changed the URI to pull these images.

```
docker pull gcr.io/cadvisor/cadvisor:v0.39.2
```

https://console.cloud.google.com/gcr/images/cadvisor/GLOBAL/cadvisor?gcrImageListsize=30

Reply:
interesting, looks like this was correct in modules but not here. will push up a change

Reply:
@alexjplant FYI